pages/cars.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div
+    v-for="car in list"
+    :key="car.id"
+    :style="{
+      'background-color': car.fields.color ? `${car.fields.color}` : 'orange',
+    }"
+    class="container"
+  >
+    <div>
+      {{ car.fields.color }}
+    </div>
+    <div>
+      {{ car.fields.Name }}
+    </div>
+    <div>
+      {{ car.fields.year }}
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'Airtable',
+  data() {
+    return {
+      list: [],
+    }
+  },
+  async created() {
```

Comment:
`created` event can't be async. Try to use `fetch` from [nuxt lifecycle](https://nuxtjs.org/docs/concepts/nuxt-lifecycle/) or `mounted` from [vue lifecycle](https://vuejs.org/api/options-lifecycle.html)

Also check errors in browser console.

Reply:
Okay, let me see

Reply:
I tried to use the fetch and mounted one after the another, both of them were not able to render the list